### PR TITLE
Add an acceptance test to cover #327

### LINKED ISSUE #327: Private key of keypair not marked as secret
### What happened?

When creating a keypair in openstack without specifying a public key it creates a public/private keypair. It seems the private key is in this case not marked as secret as it is returned in plain text when exporting as stack export.

### Expected Behavior

The private key is marked as secret in the stack.

### Steps to reproduce

Create a pulumi program like this:

```
import pulumi
import pulumi_openstack
key_pair = pulumi_openstack.compute.Keypair(
    'keypair',
    name='keypair',
    public_key=None,
)

pulumi.export('private_key', key_pair.private_key)
```

The stack output of `private_key` is not marked as secret.

### Output of `pulumi about`

```
CLI
Version      3.76.0
Go Version   go1.20.6
Go Compiler  gc

Plugins
NAME        VERSION
aws         5.42.0
github      5.14.1
kubernetes  3.28.1
openstack   3.12.1
python      unknown
random      4.13.2

Host
OS       darwin
Version  13.4.1
Arch     arm64
```

### Additional context

_No response_

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

### PR DESCRIPTION
**Pull request by t0yv0**
An upstream upgrade just fixed #327 but this PR adds an acceptance test to verify it stays fixed.

### CODE REVIEW

File: examples/keypair/index.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import * as os from "@pulumi/openstack";
+
+const keypair = new os.compute.Keypair("test", {});
+
+export const privateKey = keypair.privateKey;
```
Comment (pierskarsenbarg) [inline examples/keypair/index.ts:5]:
```suggestion
export const keypair = keypair.privateKey;
```

Comment (pierskarsenbarg) [reply examples/keypair/index.ts:5]: lol you did it the other way

**Review by aq17 — APPROVED** [review]

**Comment by t0yv0** [comment]
Having to rebase on master since it just updated upstream.. Looks like the PR is not necessary anymore because upstream fixed the problem. I can still check it in for the test coverage.

**Comment by aq17** [comment]
@t0yv0 Sorry, forgot to give you a heads up that I was about to upgrade it 😅

**Comment by t0yv0** [comment]
No problem. We'll get one more acceptance test out of this.

**Pull request merged**